src/main/resources/messages.properties

```diff
@@ -474,5 +474,10 @@ validation.date.invalid.directors_approval.date=The date cannot be on or before
 validation.characters.invalid.off_balance_sheet_arrangements.details=Enter valid characters
 validation.length.invalidInputLength.off_balance_sheet_arrangements.details=You must not exceed {1} characters
 
+# Loans To Directors
 
+loanToAdd.directorName = Director name:
+validation.element.missing.loan.director_name = Enter the director's name
 
+validation.characters.invalid.loan.director_name = Enter valid characters
```

Comment:
Where is this used? Couldn't see any validation apart from validation annotation or is the annotation doing all the work?

Reply:
This is used in the LoanValidator

Reply:
Thanks Aaron. Only asked as can only see check for if the directors name is blank and nothing about invalid characters in LoanValidator